test/github-issues/8393/issue-8393.ts

```diff
@@ -0,0 +1,59 @@
+import "reflect-metadata";
+import {
+    createTestingConnections,
+    closeTestingConnections,
+    reloadTestingDatabases
+} from "../../utils/test-utils";
+import { Connection, UpdateValuesMissingError } from "../../../src/";
+import { expect } from "chai";
+import { Post } from "./entity/Post";
+
+const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+
+describe("github issues > #8393 When trying to update `update: false` column with `@UpdateDateColumn` the update column is updated", () => {
+    let connections: Connection[];
+    before(
+        async () =>
+            (connections = await createTestingConnections({
+                entities: [__dirname + "/entity/*{.js,.ts}"],
+                schemaCreate: true,
+                dropSchema: true,
+            }))
+    );
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+
+    it("should not update the @UpdateDateColumn column when trying to update un-updatable column", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const post = new Post();
+                post.title = "Control flow based type analysis";
+                post.readOnlyColumn = 1;
+
+                await connection.manager.save(post);
+
+                await sleep(1000);
```

Comment:
why do we need to sleep here

Reply:
So the update column will changed, although it will certainly change, I can remove it or add a comment

Reply:
after `await connection.manager.save(post);` post is already updated in the db. Redundant code brings redundant questions.

Reply:
I will remove it, but just so you know (if I wasn't explaining well), this code meant to ensure that some time passes between the insertion (`save`) and the update (`update`)...